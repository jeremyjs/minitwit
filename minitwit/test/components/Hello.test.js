import React from 'react';
import Hello from '../../components/Hello'
import renderer from 'react-test-renderer';

test('Hello renders with default name', () => {
  const component = renderer.create(
    <Hello />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

test('Hello renders with name', () => {
  const component = renderer.create(
    <Hello name="Bob"/>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
