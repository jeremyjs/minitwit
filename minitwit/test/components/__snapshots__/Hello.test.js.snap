// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hello renders with default name 1`] = `
<div>
  Hello 
  David
  !
</div>
`;

exports[`Hello renders with name 1`] = `
<div>
  Hello 
  Bob
  !
</div>
`;
